app/catalog/catalog.jsx

```diff
@@ -1,8 +1,324 @@
 import { Header } from '../header';
-import { useEffect, useState } from 'react';
+import { createRef, useEffect, useState } from 'react';
+import { useLocation } from "react-router";
```

Comment:
aim for consistency with quotes


```suggestion
import { createRef, useEffect, useState } from 'react';
import { useLocation } from 'react-router';
```